ephios/plugins/qualification_management/dynamic_preferences_registry.py

```diff
@@ -0,0 +1,26 @@
+from urllib.parse import urlparse
+
+from django import forms
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+from dynamic_preferences.registries import global_preferences_registry
+from dynamic_preferences.types import StringPreference
+
+from ephios.core.dynamic_preferences_registry import general_global_section
+
+
+@global_preferences_registry.register
+class QualificationManagementReposPreference(StringPreference):
+    name = "qualification_management_repos"
+    verbose_name = _("List of qualification repositories")
+    help_text = _("To use multiple repositories, put URLs in separate lines.")
```

Comment:
help text is not rendered

Reply:
wait. yes it is!?

Reply:
![image](https://user-images.githubusercontent.com/5970076/133419355-4580c3e8-0492-42b4-8729-317e3413483e.png)